### ISSUE TITLE
Cannot resize panes with fixed position and different Layers

### ISSUE THREAD

**Issue by Kinaou** [nhold/wxWidgets#82]
When adding programmatically of several panes at the same position (in the aim of creating a notebook, eg. `paneinfo.Top()`) and by defining an incremented `Layer(i++)` value, the panes are created side by side and cannot be resized horizontally (no resizing bar).

![no_resizing_bar](https://cloud.githubusercontent.com/assets/4669963/5853946/cfe941a4-a228-11e4-870f-dce0f6ee6267.png)

**Comment by Kinaou** [comment]
What do you think about using the Layer position as an index of the notebook page (what I expected when I tried to define the layer)?

**Comment by Kinaou** [comment]
It seems to happen only when the dock direction is set to `Center()`.

**Comment by mjmacleod** [comment]
Is this specific to this branch or is this a general AUI bug that effects trunk as well?

**Comment by Kinaou** [comment]
You are right, it is not specific to this branch, that happens also on the trunk...
